frontend/src/components/common/form/FastDatePicker.test.tsx

```diff
@@ -2,7 +2,7 @@ import { FastDatePicker } from './FastDatePicker';
 import React from 'react';
 import { Formik } from 'formik';
 import { noop } from 'lodash';
-import { render, screen, wait } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
Updating the react testing-library required removing any `wait` statements.  However `waitFor` and `act` don't appear to play nice together.